core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -501,6 +501,9 @@
       create(Name.USER_FILE_BUFFER_BYTES, "8MB");
   public static final PropertyKey USER_FILE_CACHE_PARTIALLY_READ_BLOCK =
       create(Name.USER_FILE_CACHE_PARTIALLY_READ_BLOCK, true);
+  public static final PropertyKey USER_FILE_COPY_FROM_LOCAL_WRITE_LOCATION_POLICY =
```

Comment:
Can you also update the [docs](http://www.alluxio.org/docs/master/en/Configuration-Settings.html) to include this configuration?